test/integrations/testTypes.ts

```diff
@@ -101,7 +101,7 @@ export type ProcessorTestData = {
       body: ProcessorTransformationResponse[];
     };
   };
-  mockFns?: (mockAdapter: MockAdapter) => {};
+  mockFns?: (mockAdapter: MockAdapter) => void;
```

Comment:
```suggestion
  mockFns?: MockFns;
```
`MockFns` type is already present, please use it

Reply:
I've updated the mock function, so not using this anymore.